Form/Type/SeoMetadataType.php

```diff
@@ -83,4 +95,28 @@ public function getName()
     {
         return 'seo_metadata';
     }
+
+    /**
+     * @param SeoMetadata $data
+     * @return null|SeoMetadataPersistenceModel
+     */
+    private function convertToPersisterModel(SeoMetadata $data)
+    {
+        if (!class_exists('Doctrine\Bundle\PHPCRBundle\DependencyInjection\Compiler\DoctrinePhpcrMappingsPass')
```

Comment:
we use `cmf_seo.backend_type_phpcr` for the compiler pass. this is set in the DI extension. we can add a constructor parameter for the type that sets the seo metadata class default and add that parameter in the DI extension.